MetaMask/metamask-mobile
# chore: Prevent redundant Sentry sourcemap uploads

## **Description**

Made a change to `bitrise.yml` that renames the flow for builds that are made specifically for testing purposes and not for production.

The change adds an env variable that skips the creation of Sentry source map files if the env variable is found.

## **Related issues**

Fixes:

## **Manual testing steps**

1. Trigger a build that doesn't say source maps -> see source map upload is skipped
2. Trigger a build that does say source maps -> see source map upload is completed
3.

## **Screenshots/Recordings**



### **Before**

NA

### **After**

`build_android_release_with_sourcemaps`
<img width="1131" alt="Screenshot 2024-09-24 at 8 58 27 AM" src="https://github.com/user-attachments/assets/bcc80a9b-9eff-4dda-a81c-40c42b2d9148">

`build_android_release`
<img width="1129" alt="Screenshot 2024-09-24 at 8 59 28 AM" src="https://github.com/user-attachments/assets/cdb1d174-b1a8-494b-a8cb-1fb8a9cadca1">


`build_ios_release_with_sourcemaps`


`build_ios_release`
<img width="782" alt="Screenshot 2024-09-24 at 9 02 04 AM" src="https://github.com/user-attachments/assets/9b07e809-f59f-4be9-a2d4-cb84cf6c3d56">


## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
